float64 boundary
float64 state_dim
int32 hidden_neurons
float64 action_dim
float64[] limits
float64[] max_u
